config.toml

```diff
@@ -5,7 +5,7 @@ metadataformat = "yaml"
 canonifyurls = true
 
 # Enable Google Analytics by entering your tracking id
-googleAnalytics = ""
+googleAnalytics = "UA-88576792-2"
```

Comment:
this should actually be in the params section, more info here: https://gohugo.io/templates/variables/#site-variables